tools/topology/topology2/production/tplg-targets-ace3.cmake

```diff
@@ -7,4 +7,10 @@ list(APPEND TPLGS
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-ptl-rt722-4ch\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,NUM_DMICS=2,\
```

Comment:
well this is obviously wrong @bardliao @RDharageswari, please fix with NUM_DMICS=4

You'll miss me when I'm gone haha.

Reply:
> well this is obviously wrong @bardliao @RDharageswari, please fix with NUM_DMICS=4

Oops, thanks for pointing this out. @RDharageswari Can you submit a fixup patch?

> 
> You'll miss me when I'm gone haha.

Obviously yes.

Reply:
Thanks @plbossart https://github.com/thesofproject/sof/pull/9500